### DISCUSSION TITLE
Replaced inverter now have two showing in Home Assistant.

### DISCUSSION THREAD

**Discussion by myxtrastufff** [WillCodeForCats/solaredge-modbus-multi#628] [General]
I had an inverter go bad and it was replaced. In the integration I now have two of the same inverter name just different serial numbers in HA. I am using a custom energy.yaml file that does all my calculations. I see that the "new" inverter has entities all named sensor.solaredge_i3_xx_2 where the old inverter was just sensor.solaredge_i3_xx. Where xx is the rest of the entity ID. I'd rather not edit my energy.yaml but obviously can. My question is how do I get rid of the old inverter? I clicked on it and disabled it but that didn't fix my energy reporting issue because my energy.yaml doesnt account for the "_2". If I were to delete the integration and reinstall would my "new" inverter 3 come in without the "_2" at the end of the entity ID? Or is there a preferred way to remove and replace an inverter?

**Comment by myxtrastufff** [comment]
Backed up HA and then deleted the integration, rebooted and then installed the integration. Looks like everything is back to normal.  looks like the _2 at the end is now gone.

**Comment by WillCodeForCats** [comment]
You can delete individual devices in the device menu, and then new ones should be added after reloading the integration.
